urllib3/connection.py

```diff
@@ -133,11 +134,15 @@ def _new_conn(self):
             conn = connection.create_connection(
                 (self.host, self.port), self.timeout, **extra_kw)
 
-        except SocketTimeout:
+        except SocketTimeout as e:
```

Comment:
Yea why we're not using it. I think I considered merging the two before but changed my mind. The symmetry is nice, but I think I'll change this back if I remember.